junit-jupiter-api/src/main/java/org/junit/jupiter/api/Assertions.java

```diff
@@ -3448,4 +3448,48 @@ public static void assertTimeoutPreemptively(Duration timeout, Executable execut
 		return AssertTimeout.assertTimeoutPreemptively(timeout, supplier, messageSupplier);
 	}
 
+	/**
+	 * <em>Assert</em> that the supplied {@code actualValue} is an instance of the
+	 * {@code expectedType}.
+	 *
+	 * <p>Like the {@code instanceof} operator a {@code null} value is not
+	 * considered to be of the {@code expectedType} and does not pass the assertion.
+	 *
+	 * @since 5.8
+	 */
+	public static void assertInstanceOf(Class<?> expectedType, Object actualValue) {
+		AssertInstanceOf.assertInstanceOf(expectedType, actualValue);
+	}
+
+	/**
+	 * <em>Assert</em> that the supplied {@code actualValue} is an instance of the
+	 * {@code expectedType}.
+	 *
+	 * <p>Like the {@code instanceof} operator a {@code null} value is not
+	 * considered to be of the {@code expectedType} and does not pass the assertion.
+	 *
+	 * <p>Fails with the supplied failure {@code message}.
+	 *
+	 * @since 5.8
+	 */
+	public static void assertInstanceOf(Class<?> expectedType, Object actualValue, String message) {
+		AssertInstanceOf.assertInstanceOf(expectedType, actualValue, message);
+	}
+
+	/**
+	 * <em>Assert</em> that the supplied {@code actualValue} is an instance of the
+	 * {@code expectedType}.
+	 *
+	 * <p>Like the {@code instanceof} operator a {@code null} value is not
+	 * considered to be of the {@code expectedType} and does not pass the assertion.
+	 *
+	 * <p>If necessary, the failure message will be retrieved lazily from the
+	 * supplied {@code messageSupplier}.
+	 *
+	 * @since 5.8
+	 */
+	@API(status = STABLE, since = "5.8")
```

Comment:
Please mark all of the new methods "experimental".
```suggestion
	@API(status = EXPERIMENTAL, since = "5.8")
```